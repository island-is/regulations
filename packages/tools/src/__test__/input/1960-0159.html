<div class="Section1">
<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">1. gr.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Þegar hreppurinn hefur látið leggja holræsi í veg, götu eða annað opið svæði, er þeim húseigendum, er þar eiga hús að eða í námunda við og í holræsið geta náð, skylt að leggja á sinn kostnað ræsi frá húsinu, er flytji allt skolp frá því út í aðalræsið ásamt rigningarvatni, ef hægt er að koma því við. Vanræki einhver að láta gera holræsi innan hæfilegs frosts, sem hreppsnefnd setur, getur hún látið vinna verkið á kostnað húseiganda.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="IS" style="&#39mso-ansi-language:IS&#39;"> </span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">2. gr.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Öll framkvæmdastörf, sem aðalholræsi hreppsins varða, svo sem viðhald, endurbætur og umsjón alla annast hreppsnefnd eða maður, sem hún felur það. Þóknun fyrir þau störf og einnig fyrir reikningshald eða annað vegna holræsagerðarinnar ákveður hreppsnefnd.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Engum er heimilt að leggja ræsi í aðalræsi, nema að fengnu leyfi ofangreindra aðila, og er mönnum skylt að hlíta fyrirmælum þeirra um gerð og legu ræsis og annað það, er máli skiptir í sambandi við það.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="IS" style="&#39mso-ansi-language:IS&#39;"> </span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">3. gr.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Kostnaður allur við holræsakerfið greiðist úr hreppssjóði. Til þess að standast þann kostnað, ber hverjum húseiganda, sem á hús að eða í námunda við götu, veg eða opið svæði, sem holræsi er lagt í, að greiða tengigjald um leið og harm tengir ræsi við aðalræsi, er nemur 2% af fasteignamati hússins en þó ekki lægra en kr. 750.00 miðað við hús með einni íbúð. Hækkar gjald þetta um kr. 375.00 fyrir hverja íbúð í húsi umfram eina.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Nú eru hús sambyggð, en aðskilin með brunagöflum, og telst þá sérstakt hús sá hluti, sem er milli brunagafla. Ef fleiri íbúðir en ein eru í húsi og íbúðir séreign, skal heildarupphæð tengigjaldsins skiptast milli eigenda íbúðanna hlutfallslega eftir tölu íbúðanna, er þeir eiga.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Auk tengigjalds greiði hver húseigandi árlega í holræsagjald til hreppsins 15 af þúsundi af fasteignamati húsanna, en þó ekki lægra en kr. 300.00 miðað við hús með einni íbúð, og hækkar gjaldið um kr. 150.00 fyrir hverja íbúð umfram eina.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="IS" style="&#39mso-ansi-language:IS&#39;"> </span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">4. gr.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Hið árlega holræsagjald, sem og tengigjaldið, getur hreppsnefnd hækkað eða lækkað um allt að 50% án staðfestingar ráðuneytisins.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="IS" style="&#39mso-ansi-language:IS&#39;"> </span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">5. gr.</span></p>

<p><span lang="IS" style="&#39mso-ansi-language:IS&#39;">Húseigandi ber ábyrgð á greiðslu tengigjalds og holræsagjalds gagnvart hreppsnefnd. Kröfur, sem hreppsnefnd kann að öðlast samkv. 1. og 3. gr., má taka lögtaki á kostnað gjaldanda, og eru gjöld þessi tryggð með lögveði í húseigninni í næstu 2 ár eftir gjalddaga með forgangsrétti fyrir hvers konar samningsveði og aðfararveði. <span style="&#39mso-tab-count:1&#39;">     </span></span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">Gjalddagi holræsagjalds er 1. jan. ár hvert.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto&#39;"><span lang="IS" style="&#39mso-ansi-language:IS&#39;"> </span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><span lang="IS" style="&#39mso-ansi-language:IS&#39;">6. gr.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Brot gegn reglugerð þessari varða sektum allt að kr. 1000.00, nema þyngri refsing liggi við að lögum.</span></p>
</div>
<span lang="IS" style="&#39font-size:12.0pt;font-family:" times="times" new="new" times="times" new="new"><br clear="all" style="&#39page-break-before:auto;mso-break-type:section-break&#39;" />
</span>

<div class="Section2">
<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Með mál út af brotum gegn reglugerð þessari skal farið að hætti opinberra mála.</span></p>

<p style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;"><span lang="IS" style="&#39mso-ansi-language:">Reglugerð þessi, sem hreppsnefnd Mosfellshrepps hefur samið og samþykkt og gerð er samkvæmt lögum nr. 15 20. júní 1923, staðfestist hér með til að öðlast þegar gildi, og birtist til eftirbreytni öllum þeim, sem hlut eiga að máli.</span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><em><span lang="IS" style="&#39mso-ansi-language:IS&#39;"> </span></em></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><em><span lang="IS" style="&#39mso-ansi-language:IS&#39;">Félagsmálaráðuneytið, 10.</span></em> <span lang="IS" style="&#39mso-ansi-language:IS&#39;">ágúst <em>1960.</em></span></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="IS" style="&#39mso-ansi-language:IS&#39;">F. h. r.</span></strong></p>

<p align="center" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><strong><span lang="IS" style="&#39mso-ansi-language:IS&#39;">Hallgrímur Dalberg.</span></strong></p>

<p align="right" style="&#39mso-margin-top-alt:auto;mso-margin-bottom-alt:"><em><span lang="IS" style="&#39mso-ansi-language:IS&#39;">Helgi Helgason.</span></em></p>
</div>
