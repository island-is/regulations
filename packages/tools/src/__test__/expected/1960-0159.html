<h3 class="article__title">
1.
gr.
</h3>

<p>
Þegar
hreppurinn
hefur
látið
leggja
holræsi
í
veg,
götu
eða
annað
opið
svæði,
er
þeim
húseigendum,
er
þar
eiga
hús
að
eða
í
námunda
við
og
í
holræsið
geta
náð,
skylt
að
leggja
á
sinn
kostnað
ræsi
frá
húsinu,
er
flytji
allt
skolp
frá
því
út
í
aðalræsið
ásamt
rigningarvatni,
ef
hægt
er
að
koma
því
við.
Vanræki
einhver
að
láta
gera
holræsi
innan
hæfilegs
frosts,
sem
hreppsnefnd
setur,
getur
hún
látið
vinna
verkið
á
kostnað
húseiganda.
</p>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Öll
framkvæmdastörf,
sem
aðalholræsi
hreppsins
varða,
svo
sem
viðhald,
endurbætur
og
umsjón
alla
annast
hreppsnefnd
eða
maður,
sem
hún
felur
það.
Þóknun
fyrir
þau
störf
og
einnig
fyrir
reikningshald
eða
annað
vegna
holræsagerðarinnar
ákveður
hreppsnefnd.
</p>

<p>
Engum
er
heimilt
að
leggja
ræsi
í
aðalræsi,
nema
að
fengnu
leyfi
ofangreindra
aðila,
og
er
mönnum
skylt
að
hlíta
fyrirmælum
þeirra
um
gerð
og
legu
ræsis
og
annað
það,
er
máli
skiptir
í
sambandi
við
það.
</p>

<h3 class="article__title">
3.
gr.
</h3>

<p>
Kostnaður
allur
við
holræsakerfið
greiðist
úr
hreppssjóði.
Til
þess
að
standast
þann
kostnað,
ber
hverjum
húseiganda,
sem
á
hús
að
eða
í
námunda
við
götu,
veg
eða
opið
svæði,
sem
holræsi
er
lagt
í,
að
greiða
tengigjald
um
leið
og
harm
tengir
ræsi
við
aðalræsi,
er
nemur
2%
af
fasteignamati
hússins
en
þó
ekki
lægra
en
kr.
750.00
miðað
við
hús
með
einni
íbúð.
Hækkar
gjald
þetta
um
kr.
375.00
fyrir
hverja
íbúð
í
húsi
umfram
eina.
</p>

<p>
Nú
eru
hús
sambyggð,
en
aðskilin
með
brunagöflum,
og
telst
þá
sérstakt
hús
sá
hluti,
sem
er
milli
brunagafla.
Ef
fleiri
íbúðir
en
ein
eru
í
húsi
og
íbúðir
séreign,
skal
heildarupphæð
tengigjaldsins
skiptast
milli
eigenda
íbúðanna
hlutfallslega
eftir
tölu
íbúðanna,
er
þeir
eiga.
</p>

<p>
Auk
tengigjalds
greiði
hver
húseigandi
árlega
í
holræsagjald
til
hreppsins
15
af
þúsundi
af
fasteignamati
húsanna,
en
þó
ekki
lægra
en
kr.
300.00
miðað
við
hús
með
einni
íbúð,
og
hækkar
gjaldið
um
kr.
150.00
fyrir
hverja
íbúð
umfram
eina.
</p>

<h3 class="article__title">
4.
gr.
</h3>

<p>
Hið
árlega
holræsagjald,
sem
og
tengigjaldið,
getur
hreppsnefnd
hækkað
eða
lækkað
um
allt
að
50%
án
staðfestingar
ráðuneytisins.
</p>

<h3 class="article__title">
5.
gr.
</h3>

<p>
Húseigandi
ber
ábyrgð
á
greiðslu
tengigjalds
og
holræsagjalds
gagnvart
hreppsnefnd.
Kröfur,
sem
hreppsnefnd
kann
að
öðlast
samkv.
1.
og
3.
gr.,
má
taka
lögtaki
á
kostnað
gjaldanda,
og
eru
gjöld
þessi
tryggð
með
lögveði
í
húseigninni
í
næstu
2
ár
eftir
gjalddaga
með
forgangsrétti
fyrir
hvers
konar
samningsveði
og
aðfararveði.
</p>

<p>
Gjalddagi
holræsagjalds
er
1.
jan.
ár
hvert.
</p>

<h3 class="article__title">
6.
gr.
</h3>

<p>
Brot
gegn
reglugerð
þessari
varða
sektum
allt
að
kr.
1000.00,
nema
þyngri
refsing
liggi
við
að
lögum.
</p>

<p>
Með
mál
út
af
brotum
gegn
reglugerð
þessari
skal
farið
að
hætti
opinberra
mála.
</p>

<p>
Reglugerð
þessi,
sem
hreppsnefnd
Mosfellshrepps
hefur
samið
og
samþykkt
og
gerð
er
samkvæmt
lögum
nr.
15
20.
júní
1923,
staðfestist
hér
með
til
að
öðlast
þegar
gildi,
og
birtist
til
eftirbreytni
öllum
þeim,
sem
hlut
eiga
að
máli.
</p>

<p class="Dags" align="center">
<em>Félagsmálaráðuneytið,
10.</em>
ágúst
<em>1960.</em>
</p>

<p class="FHUndirskr" align="center">
<strong>F.
h.
r.</strong>
</p>

<p align="center">
<strong>Hallgrímur
Dalberg.</strong>
</p>

<p align="right">
<em>Helgi
Helgason.</em>
</p>
